schemas/observation/digital-signature.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var Joi = require( 'joi' );
+
+var signatureInfo = {
```

Comment:
Should all three properties be optional? It seems that, if you pass any of them, you should be passing all of them.